pandas/tests/arithmetic/test_numeric.py

```diff
@@ -1284,3 +1287,35 @@ def test_dataframe_div_silenced():
     )
     with tm.assert_produces_warning(None):
         pdf1.div(pdf2, fill_value=0)
+
+
+class TestNumericArraylikeArithmeticWithBool:
+    @pytest.mark.skipif(
+        compat.is_platform_32bit(),
```

Comment:
Hmm what’s this for? Should still work on 32 bit

Reply:
@WillAyd Can you please look at the reasons b38f9e6 failed.

That's what I am understanding, from looking at it.

Maybe you have other conclusions.

Reply:
I think its just a bug somewhere; we typically don't default back to platform ints

Reply:
@WillAyd Should I put a ```pytest.mark.skipif``` decorator (skiping this test, if running on 32-bit platform) at least for now?